src/app/shared/components/login-button/login-button.component.html

```diff
@@ -1,3 +1,3 @@
 <button class="button">
-    <a href="#"><img src="assets/icons/ic_person.svg" alt="img" class="icon"> Login</a>
+   <img src="assets/icons/ic_person.svg" alt="img" class="icon"> Login
```

Comment:
no need for a separate component for a simple button

Reply:
@SteveRusin login and logout buttons were moved to LoginButtonsComponent